### ISSUE TITLE
Schedulers

### ISSUE THREAD

**Issue by hbredin** [pyannote/pyannote-audio#94]
- [x] [A disciplined approach to neural network hyper-parameters: Part 1 - learning rate, batch size, momentum, and weight decay](https://arxiv.org/abs/1803.09820)
- [x] [1cycle](https://sgugger.github.io/the-1cycle-policy.html#the-1cycle-policy)
- [x] [Davis King's](http://blog.dlib.net/2018/02/automatic-learning-rate-scheduling-that.html)

**Comment by hbredin** [comment]
Those were implemented in latest release (1.0):
- `pyannote.audio.train.schedulers.DavisKingScheduler`
- `pyannote.audio.train.schedulers.CyclicScheduler` 
- `pyannote.audio.train.trainer.Trainer.auto_lr()`
